content/docs/v1.11.0/docs/antctl.md

```diff
@@ -656,7 +656,7 @@ testmulticast-vw7gx5b9 test3-sender-1               0       10
 
 ### Showing memberlist state
 
-`antctl` agent command `get memberlist` (or `get ml`) print the state of memberlist cluster of Antrea Agent.
+`antctl` agent command `get memberlist` (or `get ml`) prints the state of memberlist cluster of Antrea Agent.
```

Comment:
while this is not a big deal, it means that when you generated the docs for v1.11.0, you used the wrong version of antrea (you checked out main, instead of the v1.11.0 tag). You may want to do it again with the correct tag, to avoid confusion.

Reply:
done.